src/main/java/dev/dokan/dokan_java/examples/DirListingFileSystem.java

```diff
@@ -242,7 +241,7 @@ public int getVolumeInformation(Pointer rawVolumeNameBuffer, int rawVolumeNameSi
 
     private Path getrootedPath(WString rawPath) {
         String unixPath = rawPath.toString().replace('\\', '/');
-        String relativeUnixPath = CharMatcher.is('/').trimLeadingFrom(unixPath);
+        String relativeUnixPath = unixPath.startsWith("/")?unixPath.substring(1):unixPath;
```

Comment:
This is not equivalent:
For a path which only contains `/`, the first one returns an empty string while the latter one throws an exception.

Reply:
I see ... but just a simple think like this brings the entire Guava dependency! Guava is a very questionable library  for  modern Java ( it was good until Java 7)

Reply:
btw, maybe the entire examples/ folder need to be move to tests?